database/initializationQuery.sql

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
It is good to have 2 sql files:
- one for DB creation (DDL script)
- another for DB population with some data (either init data or sample data)